tests/coverage.sh

```diff
@@ -53,6 +67,31 @@ RUN echo "Make test gitrepo to clone from ..." && \
     git add . && \
     git commit -a -m "Initial release" && \
     git commit -a --allow-empty -m "Empty to reset to trigger update"
+
+RUN useradd -ms /bin/bash coverage
+RUN chown -R coverage:coverage /var /usr/share/git-core
+
+USER coverage
+WORKDIR /home/coverage
+RUN git config --global user.email "githook@test.com" && \
+    git config --global user.name "Githook Tests"
+
+
+## Debugging #########################################
```

Comment:
Here I left, the Debugging Code in, in case a tests fails, because kcov does not have any output -> we uncomment the kcov and run the tests and they will get copied at the end back to the host. In that way its easy to inspect the replacements.